sound/soc/sof/sof-priv.h

```diff
@@ -308,6 +308,11 @@ struct snd_sof_pcm_stream {
 	struct snd_pcm_substream *substream;
 	struct work_struct period_elapsed_work;
 	bool d0i3_compatible; /* DSP can be in D0I3 when this pcm is opened */
+	/*
+	 * flag to indicate that the DSP pipelines should be kept
+	 * active or not on the stream suspended
```

Comment:
Amend comment to say "flag to indicate that the DSP pipelines should be kept active when the stream is suspended"

Reply:
what I wanted to explain with "on the stream suspended" is "on the stream suspending", "on doing sth", sigh, I have to go back to middle school.

Reply:
Let me use "flag to indicate that the DSP pipelines should be kept active while suspending the stream" to express what I exactly wanted to.